tests/ttnn/unit_tests/operations/test_core.py

```diff
@@ -0,0 +1,106 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
Maybe name this file something better? test_core doesn't seem very descriptive to what this file is testing?

Reply:
the test name for now is consistent with the ttnn file the op is in. 
ttnn/operations/core.py

I was thinking that to this file we could add the other core operations